.github/workflows/release_snapshot.yml

```diff
@@ -0,0 +1,41 @@
+# **what?**
+# This workflow will....
+
+# **why?**
+# Because I said so
+
+# **when?**
+# This is triggered manually
+
+# **how**
+# - poorly
+
+name: Release a Snapshot
+permissions: 
+  packages: read
+  contents: write
+  pull-requests: read
+on:
+  workflow_dispatch:
+    inputs:
+      version_number:
+        description: The release version number (i.e. 1.0.0b1).
```

Comment:
Should the input have the `v` prefixed on the number or not?

Reply:
Clarified it should not include the v. Though that will not make a difference in the current implementation